### PR TITLE
build(prettier): Upgrade prettier to 1.16.4

### PR DESCRIPTION
**Pull request by billyvg**
Just taking a look at the diffs this produces

Would fix prettier for vscode (at least the latest version of the prettier extension), see https://github.com/prettier/prettier-vscode/issues/710

### CODE REVIEW

File: src/sentry/static/sentry/app/actionCreators/events.jsx [hunk 1]
```diff
@@ -105,14 +105,16 @@ export const doEventsRequest = (
...
-    absolutePeriods.filter(i => !!i).map(absolutePeriod =>
-      api.requestPromise(`${BASE_URL(organization)}`, {
-        query: {
-          ...urlQuery,
-          ...absolutePeriod,
-        },
-      })
-    )
+    absolutePeriods
+      .filter(i => !!i)
```
Comment (billyvg) [inline src/sentry/static/sentry/app/actionCreators/events.jsx]: this is better

File: src/sentry/static/sentry/app/api.jsx [hunk 2]
```diff
@@ -30,8 +30,8 @@ export function paramsToQueryArgs(params) {
   let p = params.itemIds
     ? {id: params.itemIds} // items matching array of itemids
     : params.query
-      ? {query: params.query} // items matching search query
-      : undefined; // all items
+    ? {query: params.query} // items matching search query
```
Comment (billyvg) [inline src/sentry/static/sentry/app/api.jsx]: imo worse :(

File: src/sentry/static/sentry/app/components/acl/feature.jsx [hunk 3]
```diff
@@ -141,7 +141,9 @@ class Feature extends React.Component {
     let customDisabledRender =
       renderDisabled === false
         ? false
-        : typeof renderDisabled === 'function' ? renderDisabled : () => <ComingSoon />;
+        : typeof renderDisabled === 'function'
+        ? renderDisabled
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/acl/feature.jsx]: I like this on a newline but wish it was indented.

File: src/sentry/static/sentry/app/components/activity/item.jsx [hunk 4]
```diff
@@ -79,7 +79,9 @@ class ActivityItem extends React.Component {
               orgId={orgId}
               projectId={project.slug}
               issue={issue}
-              to={`/${orgId}/${project.slug}/issues/${issue.id}/activity/#event_${item.id}`}
+              to={`/${orgId}/${project.slug}/issues/${issue.id}/activity/#event_${
+                item.id
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/activity/item.jsx]: :(

File: src/sentry/static/sentry/app/components/activity/noteInput.jsx [hunk 5]
```diff
@@ -276,9 +276,11 @@ const NoteInput = createReactClass({
 
   mentionableTeams() {
     let {group} = this.props;
-    return (ProjectsStore.getBySlug(group.project.slug) || {
-      teams: [],
-    }).teams.map(team => ({
+    return (
+      ProjectsStore.getBySlug(group.project.slug) || {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/activity/noteInput.jsx]: 👍

File: src/sentry/static/sentry/app/components/activity/noteInput.jsx [hunk 6]
```diff
@@ -276,9 +276,11 @@ const NoteInput = createReactClass({
...
+        teams: [],
+      }
+    ).teams.map(team => ({
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/activity/noteInput.jsx]: 👍

File: src/sentry/static/sentry/app/components/assigneeSelector.jsx [hunk 7]
```diff
@@ -139,7 +144,13 @@ const AssigneeSelectorComponent = createReactClass({
     this.setState({loading: true});
   },
 
-  handleAssign({value: {type, assignee}}, state, e) {
+  handleAssign(
+    {
+      value: {type, assignee},
+    },
+    state,
+    e
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/assigneeSelector.jsx]: 💩
💩
💩

File: src/sentry/static/sentry/app/components/assigneeSelector.jsx [hunk 8]
```diff
@@ -265,8 +276,9 @@ const AssigneeSelectorComponent = createReactClass({
                 <InviteMemberLink
                   data-test-id="invite-member"
                   disabled={loading}
-                  to={`/settings/${this.context.organization
-                    .slug}/members/new/?referrer=assignee_selector`}
+                  to={`/settings/${
+                    this.context.organization.slug
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/assigneeSelector.jsx]: 💩

File: tests/js/spec/views/teamMembers.spec.jsx [hunk 9]
```diff
@@ -33,8 +33,9 @@ describe('TeamMembers', function() {
   });
 
   it('can remove a team', function() {
-    let endpoint = `/organizations/${org.slug}/members/${members[0]
-      .id}/teams/${team.slug}/`;
+    let endpoint = `/organizations/${org.slug}/members/${members[0].id}/teams/${
+      team.slug
+    }/`;
```
Comment (EvanPurkhiser) [inline tests/js/spec/views/teamMembers.spec.jsx]: IMO better than breaking on the dot.

File: src/sentry/static/sentry/app/components/assigneeSelector.jsx (hunk 7 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/assigneeSelector.jsx]: It does not affect this :(

**Comment by lynnagara** [comment]
Is this still desired? Needs a rebase but otherwise seems fine to me

**Review by evanpurkhiser — COMMENTED** [review]
looks like I left a bunch of +/-1s I forgot to submit. Not that it's important.

**Comment by billyvg** [comment]
> Is this still desired? Needs a rebase but otherwise seems fine to me

Yeah I think we should still upgrade - will have to revert + re-run and try to get it in on Friday.

**Comment by EvanPurkhiser** [comment]
Lets do it

**Review by markstory — APPROVED** [review]
Mashing approve on this. Our only other option is to not upgrade which doesn't seem like a great long term solution.

**Pull request merged**